src/components/modals/WithdrawInProgressModal.vue

```diff
@@ -0,0 +1,201 @@
+ <template>
+  <modal-wrapper id="withdraw-in-progress-modal" icon="dex">
+    <div class="title">{{$t('withdrawSystemAsset', $store.state.systemWithdraw)}}</div>
+    <div class="body">
+      <div class="checklist">
+        <div class="checklist-header">{{$t('steps')}}</div>
+        <ul>
+          <li :class="stepClass(0)"><span>{{stepIndicator(0)}}</span>{{step0Label}}</li>
+          <li :class="stepClass(1)"><span>{{stepIndicator(1)}}</span>{{step1Label}}</li>
+          <li :class="stepClass(2)"><span>{{stepIndicator(2)}}</span>{{step2Label}}</li>
+          <li :class="stepClass(4)"><span>{{stepIndicator(4)}}</span>{{step4Label}}</li>
```

Comment:
Step 3 was removed due to a business decision. Leaving this as-is in case it comes back.